docs/src/pages/quick-start.md

```diff
@@ -3,7 +3,30 @@ id: quick-start
 title: Quick Start
 ---
 
-Follow the following steps to quickly monitor this Monica website and to get notification when the website is down via email using Gmail.
+## Getting Started
+
+To start monitoring URLs, you need to create a configuration file (JSON file). The configuration file contains the probes, alerts, and notification configurations. You can check the sample configuration file from our [docs](https://hyperjumptech.github.io/monika/quick-start) or copy from one of these files:
```

Comment:
We can remove this 

You can check the sample configuration file from our [docs](https://hyperjumptech.github.io/monika/quick-start)

because this page is the quick start page